libraries/resource_hab_sup.rb

```diff
@@ -35,9 +35,12 @@ class HabSup < Resource
       property :hab_channel, String
       property :auto_update, [true, false], default: false
       property :auth_token, String
+      property :license, String
```

Comment:
We should perform validation on all of these by making sure they're only the valid license strings.

Reply:
Validation added!